okrs.md

```diff
@@ -0,0 +1,45 @@
+# Objectives and Key Results (OKRs)
+
+Hypha uses Objectives and Key Results for planning and setting collective goals.
+We currently define OKRs at the working group level and update them quarterly in our [**OKR tracking sheet**](https://docs.google.com/spreadsheets/d/1hMFS3IhzZOFQA-yjt7lFcoVO0Ry6erFEH7D1NswvDtA/edit). 
+
+## About OKRs
+
+_Adapted from [Trello blog](https://blog.trello.com/okrs-set-achieve-track-trello) shared by patcon in chat._
```

Comment:
A little too much information to reference individual sharing in chat for handbook.

```suggestion
_Adapted from [Trello blog](https://blog.trello.com/okrs-set-achieve-track-trello)._
```